storybookjs/storybook
# Fix/angular cli config

Issue:

## What I did

`outputPath`, `scripts`, `styles` are required by `@angular-devkit/build-angular`, make sure it won't break if they are not presented.

## How to test

- Is this testable with Jest or Chromatic screenshots?
Yes, added.

- Does this need a new example in the kitchen sink apps?
No need.

- Does this need an update to the documentation?
No need.

If your answer is yes to any of these, please make sure to include it in your PR.
